patches/rules_jvm_external.default_public_visibility.patch

```diff
@@ -0,0 +1,13 @@
+diff --git coursier.bzl coursier.bzl
```

Comment:
this looks like a git diff, do we need this? I see it used[ here ](https://github.com/player-ui/player/pull/256/files#diff-5493ff8e9397811510e780de47c57abb70137f1afe85d1519130dc3679d60ce5R20) but why

Reply:
`rules_jvm_external` made the change to not make the downloaded artifacts public:
https://github.com/bazelbuild/rules_jvm_external/pull/649

We only rely on the artifacts directly for `kotlinx_coroutines_core_fixed`, which I hope will go away at some point. So, rather than push a commit to change the behavior in `rules_jvm_external`, we apply a patch on demand to fix the issue in our workspace.

Reply:
oh wow so these changes are actually taking effect when you use the repo at specified branch? That's pretty cool but at the same time it's kinda nuts this capability exists

Reply:
Yeah, this patch would be applied to the repo after it's cloned. Definitely a pretty "dangerous" feature. It exists for other types of repo declarations as well, like `http_archive`:
https://bazel.build/rules/lib/repo/http#http_archive-patches